error_test.go

```diff
@@ -152,6 +168,12 @@ func assertAPIErrorMessage(t *testing.T, apiErr APIError, expected string) {
 	}
 }
 
+func assertAPIErrorInnerError(t *testing.T, apiErr APIError, expected interface{}) {
+	if apiErr.InnerError != expected {
```

Comment:
@ZeroDeng01  Please use reflect.DeepEqual as you want to compare all the fields of the struct.

Reply:
@vvatanabe THANKS！Modifications have been made, but not sure if I did it right, this one needs your help to review.👍

```golang
func assertAPIErrorInnerError(t *testing.T, apiErr APIError, expected interface{}) {
	if !reflect.DeepEqual(apiErr.InnerError, expected) {
		t.Errorf("Unexpected APIError InnerError: %v; expected: %v; ", apiErr, expected)
	}
}
```